aries-site/src/pages/foundation/tshirt-sizing.mdx

```diff
@@ -0,0 +1,159 @@
+import { Box, Image } from 'grommet';
+import { Example } from '../../layouts';
+import { ButtonSizingExample } from '../../examples';
+
+## What is t-shirt sizing?
+
+T-shirt sizing is the way that Grommet leverages familiar clothing sizes of "small", "medium", "large", and other ranges of sizes to drive the sizing and spacing of components. Grommet maps these t-shirt sizes to specific pixel values. The derivation of these pixel values will be explained more below.
+
+T-shirt sizing controls two aspects of a UI: the width/height of components and the spacing within and around components (margin, pad, border, etc.)
+
+When designing and developing with the HPE Design System, you should always use t-shirt sizes. If you find yourself adding custom px values to any properties, this should be a warning that you need to consult with your designer about why a custom value is needed. In nearly every case, designs and developement should be using t-shirt sizes.
+
+## Why do we use t-shirt sizing?
+
+Using t-shirt sizes results in a UI that is more consistent across pages and across applications, creating a more unified experience for HPE customers.
+
+There are also benefits of t-shirt sizing for designers and developers!
+
+Using pixel values, percentages, or other common sizing methods to create layouts can result in applications that are less composable and less scalable. T-shirt sizing creates a common language between designers and developers and defined intervals of sizes that makes designing and developing more efficient. Instead, a container of content may just have of a width of "medium", for example, and Grommet will take care of the rest.
+
+## The base unit
+
+In order to create the range of sizes and their corresponding pixel values, a base unit of `24px` is used as a scalar.
+
+Below is a snippet from Grommet's [`base.js`](https://github.com/grommet/grommet/blob/master/src/js/themes/base.js) theme file where the calculations and corresponding pixel values for the t-shirt sizes that drive the width/height of components such as Box are defined. The `base.js` theme establishes many of the defaults that are leveraged across Grommet applications.
+
+```
+// base.js in Grommet
+
+xxsmall: `${baseSpacing * 2}px`, // 48
+xsmall: `${baseSpacing * 4}px`, // 96
+small: `${baseSpacing * 8}px`, // 192
+medium: `${baseSpacing * 16}px`, // 384
+large: `${baseSpacing * 32}px`, // 768
+xlarge: `${baseSpacing * 48}px`, // 1152
+xxlarge: `${baseSpacing * 64}px`, // 1536
+full: '100%',
+```
+
+## T-shirt sizing for component dimensions
+
+One of the main ways t-shirt sizing is used is to control the width/height of a component. For some components, like [Box](/components/box), this can be controlled with the `width` and `height` props. For other components, like [Avatar](https://v2.grommet.io/avatar?theme=hpe), [Table](/components/table), or [Text](/foundation/typography#text), the dimensions are controlled by the `size` prop.
```

Comment:
```suggestion
One of the main ways t-shirt sizing is used is to control the width/height of a component. For some components, like [Box](/components/box), this can be controlled with the `width` and `height` props. For other components, like [Avatar](https://v2.grommet.io/avatar?theme=hpe), [Table](/components/table), or [Text](/foundation/typography#text), the dimensions are controlled by the `size` prop. And others, like [Grid](/components/grid), t-shirt sizes specify `row` and `colum` dimensions.
```